app/Http/Controllers/Secretariat/UserController.php

```diff
@@ -115,8 +114,7 @@ public function setCollegistType(Request $request)
     public function list()
     {
         $this->authorize('viewAny', User::class);
-        $users = User::role(Role::COLLEGIST)
-            ->with(['roles', 'workshops', 'educationalInformation', 'allSemesters'])->orderBy('name')->get();
+        $users = User::all()->with(['roles', 'workshops', 'educationalInformation', 'allSemesters'])->orderBy('name')->get();
```

Comment:
This is not right, if we want to get all users, just leave the `all()` call.
See the difference is that before calling `all`, `get`, `first`, `last`, etc. we have a Builder instance. After, that, we get a collection instance. For query Builders we can say which columns we want to include in the query, but for collections it's too late.

Also, I am not sure if we want to include all users here. Maybe having one for the secretary and having one for the dormitory staff would make sense? I think collegists and tenants should be handled differently.

Reply:
Okay, I will restore the original one.